### PR TITLE
Holotablets

### PR DESCRIPTION
**Pull request by ghost**
Adds another type of modular computer: holotablets.
They're meant to be used as a replacement for paper (reports, clipboards, this stuff. also, e-reading) - no network and additional hardware. Available only in games vending for now.. might be worth adding them into fabricators?

It's just fancy and useless, that's all. Of course you can read books and fill out reports on your PDA.. But hey, it glows and has big screen!

image
^ all of the types for now (they also have powered down state, and the "screen" is animated with scanline)
image
^ light power

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Not sure how much I like those icons, to be honest.

**Comment by ghost** [comment]
me neither, the curvy one (purple) seems the most odd
idk? i'm not a spriter anyway, tbh the handle of second tablet is made from holopad ;d

**Comment by ghost** [comment]
hm that is strange it fails 
seems that there is no assembly extension

**Review by MistakeNot4892 — DISMISSED** [review]

File: code/modules/modular_computers/computers/subtypes/dev_holo.dm [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+/obj/item/modular_computer/holotablet/on_update_icon()
+	overlays.Cut()
+	var/datum/extension/assembly/modular_computer/assembly = get_extension(src, /datum/extension/assembly)
+	if(assembly && assembly.enabled)
+		var/image/A = image(icon,"[icon_state]-on")
+		A.color     = display_color
+		overlays   += A
+
+		var/icon/I = icon(icon,"[icon_state]-screen")
```
Comment (comma) [inline code/modules/modular_computers/computers/subtypes/dev_holo.dm:66]: ew icon procs
should at least cache these

Comment (ghost) [reply code/modules/modular_computers/computers/subtypes/dev_holo.dm:66]: will resolve this, sry

**Pull request closed without merging**